src/Room.js

```diff
@@ -0,0 +1,13 @@
+class Room {
```

Comment:
I've seen some people make a Room class - do you need that class if you aren't instantiating new Rooms?